spec/features/dams_object_spec.rb

```diff
@@ -505,4 +505,9 @@
     expect(page).to have_selector('button#node-btn-1:first',:text=>'Parameters')
     page.should_not have_css("button#node-btn-1", :count => 2)
   end
+  it "should not see label Creation Date and Issued Date" do
```

Comment:
Good catch. I've corrected it to "should display". See https://github.com/ucsdlib/damspas/commit/34cefa16a526f05a42aaf9e082abaf223177a891.